app/javascript/src/components/editor/ExpandableSnippet.svelte

```diff
@@ -33,13 +34,26 @@
 
   $: syntaxHighlight(expanded)
 
+  async function scrollToHighlightedLine() {
+    await tick()
+
+    if (!expandedCodeEl) return
+
+    const highlightedLineEl = expandedCodeEl.querySelector(".expandable-snippet__line--highlighted")
+    if (!highlightedLineEl) return
+
+    highlightedLineEl.scrollIntoViewIfNeeded()
+  }
+
+  $: if (expanded) scrollToHighlightedLine()
```

Comment:
Would be nice if this was moved up to be with the other reactive functions. It doesn't have to be declared after the function, it gets hoisted anyway.

Reply:
It was more about keeping related things close to each other for readability, but sure 👍